readmes/python-repos-setup.md

```diff
@@ -65,4 +80,14 @@ Then run gunicorn using the following command:
 
 ```bash
     gunicorn wsgi:app -c run/gunicorn/local.py
-```
\ No newline at end of file
+```
+
+## Extras
```

Comment:
This bits already covered here
https://github.com/communitiesuk/funding-service-design-workflows/blob/main/readmes/python-repos-ide-setup.md

Reply:
Good Catch